docs/resources/cci_network.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Cloud Container Instance (CCI)"
+---
+
+# huaweicloud_cci_network
+
+Manages a CCI Network resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "namespace_name" {}
+variable "network_name" {}
+variable "network_id" {}
+variable "security_group_id" {}
+
+data "huaweicloud_availability_zones" "test" {}
+
+resource "huaweicloud_cci_network" "test" {
+  availability_zone = data.huaweicloud_availability_zones.test.names[0]
+  namespace         = var.namespace_name
+  name              = var.network_name
+  network_id        = var.network_id
```

Comment:
the `var.network_name` is confused with network_id

Reply:
Already use `vpc_network_id` instead of `network_id`.